web/app/js/src/jsoneditor-loader.js

```diff
@@ -25,16 +30,19 @@ $(document).on('click', '.jsonLoader', function(){
 });
 
 $(document).on('click', '.jsonSaver', function(){
-  $.post('app/controllers/EntityAjaxController.php', {
-    updateJsonFile: true,
-    id:currentFileUploadID,
-    upload_id: currentFileItemID,
-    type: currentFileType,
-    content: JSON.stringify(editor.get())
-  }).done(function(data){
-    notif(data);
-    if(data.msg==='JSON file updated successfully'){
-      $('.jsonSaver').attr('disabled', 1).text('Saved').css('cursor','default');
-    }
-  });
+  var formData = new FormData();
+  var blob = new Blob([JSON.stringify(editor.get())], { type: 'application/json' });
```

Comment:
can you explain the Blob use?

Reply:
I am using this because the php code that handles the request requires it to be a File object.